Themis/REST/RESTController.swift

```diff
@@ -57,7 +57,13 @@ class RESTController {
             Authentication.shared.authenticated = false
             throw RESTError.unauthorized
         case 403:
-            throw RESTError.forbidden
+            guard let data = data,
+                  let json = try? JSONSerialization.jsonObject(with: data, options: .allowFragments),
+                    let jsonDict = json as? [String: Any], let detail = jsonDict["detail"] as? String else {
+                
+                throw RESTError.forbidden
+            }
+            throw CustomError.error(title: RESTError.forbidden.errorDescription ?? "Test", description: detail)
```

Comment:
Maybe we should just try to parse the server message for any HTTP status code instead of just for 403? Of course, there should still be some default when the server does not send a message.

Reply:
So far I've only looked into the case when it is forbidden, I've not checked if the error message is accessible under "detail" for other errors too.